docs/how-to/installation/setup-gke-cluster.txt

```diff
@@ -75,6 +75,43 @@ their local machine.
    # Create a GCS bucket to store checkpoints.
    gsutil mb gs://${GCS_BUCKET_NAME}
 
+########################
+ Managing a GKE Cluster
+########################
+
+*************
+ Node Taints
+*************
+
+For general instructions on adding taints and tolerations to nodes,
+please see the :ref:`Taints and Tolerations <taints-on-kubernetes>`
+section in our :ref:`Guide to Kubernetes <install-on-kubernetes>`.
+There, you can find an explanation of taints and tolerations, as well as
+instructions for using ``kubectl`` to add them to existing clusters.
+
+It is important to note that if you use the GCP CLI to add taints to
+nodes, you must also add tolerations using ``kubectl``; otherwise,
+Kubernetes will be unable to schedule pods on the tainted node.
+
+To create a nodepool or a cluster with a taint in GKE, use the
+`--node-taints` flag to specify the type, tag, and effect.
+
+.. code:: bash
+
+   gcloud container clusters create ${GKE_CLUSTER_NAME} \
+      --node-taints ${TAINT_TYPE}=${TAINT_TAG}:${TAINT_EFFECT}
+
+   gcloud container clusters create ${GKE_CLUSTER_NAME} \
+      --node-taints dedicated=experimental:PreferNoSchedule
+
+   gcloud container node-pools create ${GKE_NODE_POOL_NAME} \
+      --cluster ${GKE_CLUSTER_NAME} \
+      --node-taints ${TAINT_TYPE}=${TAINT_TAG}:${TAINT_EFFECT}
+
+   gcloud container node-pools create ${GKE_NODE_POOL_NAME} \
+      --cluster ${GKE_CLUSTER_NAME} \
+      --node-taints special=gpu:NoExecute
```

Comment:
same as above, some additional explanation or "i.e." stuff would be nice. This is also stuff that is documented in kubernetes, so it might even be sufficient just to link those docs.